pkg/strategy/xdepthmaker/metrics.go

```diff
@@ -0,0 +1,107 @@
+package xdepthmaker
+
+import (
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var spreadRatioMetrics = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "bbgo_xdepthmaker_market_spread_ratio",
+		Help: "the spread ratio of the market",
+	},
+	[]string{"strategy_type", "strategy_id", "exchange", "symbol"},
+)
+
+var openOrderMaxPriceSpacingMetrics = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "bbgo_xdepthmaker_open_order_max_price_spacing",
+		Help: "the max price spacing of open orders on the market",
+	},
+	[]string{"strategy_type", "strategy_id", "exchange", "side", "symbol"},
+)
+
+var openOrdersCountMetrics = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "bbgo_xdepthmaker_market_open_order_count",
```

Comment:
can be just "bbgo_xdepthmaker_open_order_count"?

Reply:
Sure.